run_server.py

```diff
@@ -2,9 +2,20 @@
 # uses wsgi to start up, and bypasses this file. So we are free to have debug
 # settings enabled.
 from Donut import app, config
+import argparse
+
+parser = argparse.ArgumentParser(description='Set up development server.')
+parser.add_argument('-p', '--port', metavar='port', type=int, 
+                    help='Port on which to run server.')
+
+args = parser.parse_args()
+
 
 if __name__ == "__main__":
-    port = getattr(config, 'PORT', 5050)
+    if args.port:
```

Comment:
Testing if a value is not none should always be done with `if args.port is not None`, since that makes it unambiguous. If I passed in `--port 0` as the arg, this would evaluate to false incorrectly.